src/LoRa.cpp

```diff
@@ -607,9 +607,21 @@ void LoRaClass::setOCP(uint8_t mA)
   writeRegister(REG_OCP, 0x20 | (0x1F & ocpTrim));
 }
 
+void LoRaClass::LoRandomSeed() {
+  writeRegister(REG_OP_MODE, 0x72);
+  writeRegister(REG_MODEM_CONFIG_1, 0x72);
```

Comment:
Could this re-use existing API's that set `REG_MODEM_CONFIG_1` and `REG_MODEM_CONFIG_2`? See the notes in the example sketch on what radio settings are used :)

Reply:
Sure. I tried to make LoRandom.h compatible with any LoRa lib, so we couldn't expect to have existing presets. But within LoRa.h sure.

Reply:
@sandeepmistry I do not understand this well, can you give me an example?

Reply:
If you look at the original sketch by @Kongduino: https://github.com/sandeepmistry/arduino-LoRa/blob/4d8f70036850128ffb52ec0b3de8ab7dfe7f1b39/examples/LoRandom/LoRandom.ino#L10-L18

```
/*
  LoRandomSeed
  Example demonstrating how to generate random numbers using LoRa.
  created 7 Sep 2020
  by Konduino
  adapted by Andres Sabas
  
  SX1276 Register (address)     Register bit field (bit #)      Values    Note
  RegOpMode (0x01)              LongRangeMode[7]                ‘1’       LoRa mode enable
                                Mode[2:0]                       ‘101’     Receive Continuous mode
  ------------------------------------------------------------------------------------------------------------------
  RegModemConfig1 (0x1D)        Bw[7:4]                         ‘0111’    ‘0111’ for 125kHz modulation Bandwidth
                                CodingRate[3:1]                 ‘001’     4/5 error coding rate
                                ImplicitHeaderModeOn[0]         ‘0’       Packets have up-front header
  ------------------------------------------------------------------------------------------------------------------
  RegModemConfig2 (0x1E)        SpreadingFactor[7:4]            ‘0111’    ‘0111’ (SF7) = 6kbit/s
  To generate an N bit random number, perform N read operation of the register RegRssiWideband (address 0x2c)
  and use the LSB of the fetched value. The value from RegRssiWideband is derived from a wideband (4MHz) signal strength
  at the receiver input and the LSB of this value constantly and randomly changes.
*/
```

You can map most of the register values to:

```
setSignalBandwidth(125E3);
setCodingRate4(5);
setSpreadingFactor(7);
```

Implicit header and continuous receive mode need to be set via new methods however.

Reply:
I have added a new continuosMode and re-use existing API's